rtrlib/lib/vector.h

```diff
@@ -0,0 +1,94 @@
+/*
+* This file is part of RTRlib.
+*
+* This file is subject to the terms and conditions of the MIT license.
+* See the file LICENSE in the top level directory for more details.
+*
+* Website: http://rtrlib.realmv6.org/
+*/
+
+#ifndef LRTR_VECTOR_UTILS_H
+#define LRTR_VECTOR_UTILS_H
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+/**
+ * @brief ASPA Object
+ * here the customer as number (CAS) authorizes a set of provider AS numbers
+ */
+struct aspas_record {
+	uint32_t customer_as_number;
```

Comment:
I'd recommend calling that member `customer_asn` (in accordance with packet pdu member names)

Reply:
done